.eslintignore

```diff
@@ -9,4 +9,5 @@ coverage
 *.scss
 **/out/*
 next-env.d.ts
-next.config.js
\ No newline at end of file
+next.config.js
+src/stories/*
```

Comment:
Why no lint checks for our storybook?

Reply:
I don't really think it's necessary there and the biggest culprit triggering them right now is the console.logs. Could probably change it, if we need it there.

Reply:
I went ahead and changed it